src/main/java/com/api/Timesheets/services/UserServiceImpl.java

```diff
@@ -144,4 +145,29 @@ public void createUser(UserDTO userDTO){
         userDao.save(user);
 
     }
+
+    @Override
+    public void requestAccess(UserDTO userDTO) {
+        User user = User.fromUserDTO(userDTO);
+        user.setActive(false);
+        userDao.save(user);
+        String adminEmail = userDao.findAll().get(0).getEmail();
+        try {
+            ImmutableMap<String, Object> model = ImmutableMap.of(
+                    "user", user,
+                    "expirationDays", tokenExpirationDays,
+                    "appUrl", appUrl);
+            String html = modelToHtmlConverter.convert(REQUEST_ACCESS_USER_HTML_TEMPLATE, model);
```

Comment:
idi no file found exception ythrow chestadi if you dont have a template file in resources. Best remove this convert part